src/layouts/docs/partials/sidebar-sidecar-layout/style.module.css

```diff
@@ -0,0 +1,65 @@
+/*
+NOTE:
+Relies on `--sticky-bars-height` being set,
+this is currently being done in `BaseNewLayout`.
+See comment in src/layouts/base-new/base-new-layout.module.css
+for further details.
+*/
+
+.contentWrapper {
+  display: flex;
+  /* flex-grow ensures the content fills available
+  space in BaseNewLayout */
+  flex-grow: 1;
+
+  /* For a coherent visual effect, the top padding value
+  must be used for sticky positioning in addition to
+  padding the content area. So, we create a CSS var
+  to make it reusable and express this intent. */
+  --main-area-top-padding: 32px;
+}
+
+.sidebar {
+  background-color: white;
+  box-shadow: 0 2px 3px rgba(101, 106, 118, 0.1),
+    0 12px 28px rgba(101, 106, 118, 0.25);
```

Comment:
I think they do have something to do with the layout - the `sidebar` part of the layout creates a scroll container and positions itself to take up the expected height portion of the screen, even if its contents aren't that tall. The shadow etc need to be applied at that level, I think, rather than within the scroll container. Moving the `box-shadow` & `background-color` into `components/sidebar` would mean we'd need to break that model, ie rather than just having the sidebar contents, `components/sidebar` would then need to take on some of these other layout concern styles as well (`max-height`, `width`, etc).

Let me know if that layout vs content rationale seems reasonable 🙇‍♂️

Reply:
I considered the `background-color` and `box-shadow` to be properties that are set regardless of the sidebar's positioning in the layout, things that are more "constant" across usage rather than dependent on the layout. Thinking some more on it `box-shadow`, would be different if in some future layout we put the sidebar on the right side of the page, so that probably is layout dependent! `background-color` seems like a constant though, and like something that should be delegated to the sidebar component to set separate from the layout.

Reply:
@ashleemboyer Sounds good, thanks for the added thoughts - I've moved `background-color` to `sidebar` 👍

Reply:
Awesome! I'll mark this as resolved 😊